src/Test.UnitTests.Sarif.Driver/Sdk/DriverSdkExtensionsTests.cs

```diff
@@ -31,7 +31,7 @@ public void ConstructUriBaseIdsDictionary_RetrievesAbsoluteFilePath()
             uriBaseIds.Values.First().Uri.LocalPath.Should().Be(path);
         }
 
-        [Fact]
+        [Fact(Skip = "If run concurrently with tests for other libraries, this test fails. To be investigated.")]
```

Comment:
> [Fact(Skip = "If run concurrently with tests for other libraries, this test fails. To be investigated.")] [](start = 7, length = 106)

opened issue #213 for tracking this seperately. I tried the .substring trick but it didnt work intermittently.